integration/js/utils/SdkBaseTest.js

```diff
@@ -300,6 +299,11 @@ class SdkBaseTest extends KiteBaseTest {
         return 'meeting'
     };
   }
+
+  getTransformedURL (url, key, value) {
```

Comment:
Checks are failing because as we need to have `this.getTransformedURL` where earlier it is just `getTransformedURL`.

Reply:
Free functions are better than methods every time :)

Reply:
thanks, updated